.github/actions/test-data-plane/action.yaml

```diff
@@ -0,0 +1,57 @@
+name: 'Test Data Plane'
+description: 'Runs tests on the Pinecone data plane'
+
+# - pod vs serverless
+# - grpc vs rest
+# - metric -> vector vs sparse vector
+# - namespace: default vs custom
+# - environment: free vs paid
+# - with metadata vs without metadata
+
+inputs:
+  metric:
+    description: 'The metric of the index'
+    required: true
+  spec:
+    description: 'The deploy spec of the index'
+    required: true
+  use_grpc:
+    description: 'Whether to use gRPC or REST'
+    required: true
+  freshness_timeout_seconds:
+    description: 'The number of seconds to wait for the index to become fresh'
+    required: false
+    default: '60'
+  PINECONE_API_KEY:
+    description: 'The Pinecone API key'
+    required: true
+
+outputs:
+  index_name:
+    description: 'The name of the index, including randomized suffix'
+    value: ${{ steps.create-index.outputs.index_name }}
```

Comment:
Is this referring to a different `create-index` step that's present in this action, or is it assuming it's been run prior? It's working properly so I'm just going to assume the scope for `steps.<whatever>` is global to the whole workflow. 😅

Reply:
I think this `outputs` declaration is just accidental copy/pasta from the `create-index` action I made earlier this week for dependency testing purposes so it can be removed. If I wanted to use it I could by having my test suite write the name of the index it creates to the `GITHUB_OUTPUT` location (see [here](https://github.com/pinecone-io/pinecone-python-client/blob/spruce/scripts/create.py#L15) in script used by create-index action) but in this case it's not needed since the test creates and deletes the index.